test/whitelisted_crowdsale.js

```diff
@@ -53,22 +46,62 @@ contract('WhitelistedCrowdsale', async (accounts) => {
       await instanceWhitelistedCrowdsale.endTime.call(), endTime
     );
     assert.equal(
-      await instanceWhitelistedCrowdsale.rate.call(), rate
+      await instanceWhitelistedCrowdsale.rate.call(), defaultRateForPrivateOffer
     );
     assert.equal(
-      await instanceWhitelistedCrowdsale.minPurchase.call(), minPurchase
+      await instanceWhitelistedCrowdsale.minPurchase.call(), defaultMinPurchaseForPrivateOffer
     );
     assert.equal(
-      await instanceWhitelistedCrowdsale.maxPurchase.call(), maxPurchase
+      await instanceWhitelistedCrowdsale.maxPurchase.call(), defaultMaxPurchaseForPrivateOffer
     );
     assert.equal(
-      await instanceWhitelistedCrowdsale.wallet.call(), wallet
+      await instanceWhitelistedCrowdsale.wallet.call(), escrowAccount
     );
     assert.equal(
       await instanceWhitelistedCrowdsale.token.call(), token.address
     );
     assert.equal(
-      await instanceWhitelistedCrowdsale.countPurchaseAmount.call(), countPurchaseAmount
+      await instanceWhitelistedCrowdsale.countPurchaseAmount.call(), defaultCountPurchaseAmountForPrivateOffer
+    );
+  });
+
+  it("should throw error when the beneficiary not in whitelist", async () => {
+    try {
+      await instanceWhitelistedCrowdsale.buyTokens.sendTransaction(userNotFromWhitelist, {from: owner, value: web3.toWei(2, "ether")});
```

Comment:
Why do we send `from: owner` here (and in other tests)? Shouldn't it be `from: userNotFromWhilelist`?